src/Lexer.php

```diff
@@ -352,4 +375,24 @@ public static function decodeJson($json)
     {
         return json_decode($json, true);
     }
+
+    /**
+     * Escape plain text output before mixing in a html context.
+     * 
+     * This should be used on any input or attributes in a delta operation.
+     * For escaping input, use Line->getInput() instead as it keeps track of only doing it once for nested inline listeners.
+     * 
+     * @since 1.2.0
+     * 
+     * @param  string $value
+     * @return string
+     */
+    public function escape($value)
+    {
+        if ($this->escapeInput === false) {
+            return $value;
+        }
+        
+        return htmlspecialchars($value, $this->escapeFlags, $this->escapeEncoding, $double=false);
```

Comment:
I assume with `$double=false` you want to make clear, this is disabled. You can keep it if you like but we should add this to the phpdoc section as this is what people would see if we use api generators. Please use `$double = false` with space, to be consistent with variable defintion.

Reply:
:+1: explain why it needs to be there and what is the difference between `$double` and `$isEscaped`.